### DISCUSSION TITLE
Returned Currency in Trading API response.

### DISCUSSION THREAD

**Discussion by Ro-Knight** [cloudbet/docs#48] [General]
Hi CloudBet team,

We were curious to know, when we place a bet with the Trading API using a currency of our choice for example USD, the accepted response we get back has a stake in it. Will it be the in the same currency that was used originally in the request, such as USD? 

As for now we have been testing with PLAY_EUR only, but we wanted to make sure what is the stake currency we get back on the response upon placing a bet.

This is also most important in case when we get back a STAKE_ABOVE_MAX or PRICE_ABOVE_MARKET or LIABILITY_LIMIT_EXCEEDED. Should we assume that the response will always contain the stake in the same currencies that was set in the request?

**Comment by kgravenreuth** [comment]
Hi @Ro-Knight 

Thanks for raising this issue. ~The `stake` would be returned in EUR.~ The `stake` is returned in the currency in which the bet was placed. We will clarify the response payload docs further for this soon.

Let me know if you see any other issues.

Cheers,
Klaus Gravenreuth
Product & Engineering @ Cloudbet

**Comment by kgravenreuth** [comment]
@Ro-Knight apologies for the confusion, please see my update above for clarification.
